doc/languages-frameworks/go.xml

```diff
@@ -68,12 +68,12 @@ pet = buildGoModule rec {
   </para>
 
   <para>
-    <varname>modSha256</varname> can also take <varname>null</varname> as an input.
+    <varname>vendorSha256</varname> can also take <varname>null</varname> as an input.
 
-    When `null` is used as a value, the derivation won't be a
-    fixed-output derivation but disable the build sandbox instead. This can be useful outside
-    of nixpkgs where re-generating the modSha256 on each mod.sum changes is cumbersome,
-    but will fail to build by Hydra, as builds with a disabled sandbox are discouraged.
+    When `null` is used as a value, rather than fetching the dependencies
+    and vendoring them, we use the vendoring included within the source repo.
+    If you'd like to not have to update this field on dependency changes, 
```

Comment:
I'm not sure this should be part of the manual since my feeling is that vendoring deps is not a good practice.

Reply:
We should say more prominently that this is discouraged, but IMHO, we should document all the features available.

Reply:
I don't agree - for the longest time vendoring deps in go was the *only* way to build safely and cleanly. If you're primarily building with nix - vendoring your deps is likely still the cleanest way to build anything (since you only need one source bundle).

To this day, if I was running a go project (and I have in the past), go generate and go build are run at commit time to ensure that all files are generated + vendored, so that you get nice stable builds. (go mod maybe replaces the vendor command, but nix apparently hates it so that's not really an option for nix users).